uat/codestyle/findbugs-exclude.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
delete the whole codestyle directory. Reference the pmd and other rules you need using `../`

Reply:
removed in 27fbe2119408cac23cefed5f76158c09e8a3cc20